.travis.yml

```diff
@@ -0,0 +1,54 @@
+# Configuration file for Travis continuous integration.
+# See https://travis-ci.org/apache/calcite-avatica-go
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to you under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+language: go
+
+branches:
+  only:
+    - master
+    - /^branch-.*$/
+    - /^[0-9]+-.*$/
+
+go:
+  - "1.10.x"
+
+sudo: required
+services:
+  - docker
+
+env:
+  global:
+  - AVATICA_IMAGE=boostport/hbase-phoenix-all-in-one:1.3-4.13
+  - AVATICA_HOST=http://localhost:8765
+
+before_install:
+  - go get -u github.com/golang/dep/cmd/dep
+  - dep ensure -v
+  - docker pull $AVATICA_IMAGE
+  - docker run -d -p 8765:8765 $AVATICA_IMAGE
+  - docker ps -a
+
+install:
+  - go build
+
+script:
+  - go test -cover -v $(go list ./... | grep -v /vendor/)
```

Comment:
We can now use `go test -cover -v ./...` because newer versions of Go will no longer test repos in `/vendor/`.